packages/app-store/routing-forms/pages/route-builder/[...appPages].tsx

```diff
@@ -778,7 +852,7 @@ export default function RouteBuilder({
       form={form}
       appUrl={appUrl}
       enrichedWithUserProfileForm={enrichedWithUserProfileForm}
-      Page={({ hookForm, form }) => {
+      Page={function Page({ hookForm, form }) {
```

Comment:
Eslint complained that useLocale can't be used in it as it though it isn't a component